### PR TITLE
[Feature][Dev] Translate Microbench To English

### PR DESCRIPTION
**Pull request by Segun-Ogundipe**
# What is the purpose of the pull request

This PR adds the English translation of microbench documentation to the website.

# Change Log

- Add "docs/en-us/development/microbench.md"
- Modified "site_config/development.js"

# How to verify this PR

- Clone the branch
- npm i
- docsite start
- Go to "http://localhost:8080/en-us/docs/development/microbench.html"

### CODE REVIEW

File: site_config/development.js [hunk 1]
```diff
@@ -1,163 +1,167 @@
 export default {
-    'en-us': {
-        sidemenu: [
-            {
-                title: 'Contribution Guide',
-                children: [
-                    {
-                        title: 'how to become a committer',
-                        link: '/en-us/docs/development/become a committer.html',
-                    },
```
Comment (CalvinKirs) [inline site_config/development.js:10]: It is recommended not to change these, especially in terms of code

Comment (Segun-Ogundipe) [reply site_config/development.js:10]: I only added a new entry for microbench link as was documented in the README.md file. My linter formatted the file and that's why the whole file shows to be edited.

**Comment by CalvinKirs** [comment]
@Tianqi-Dotes PTAL?

**Comment by CalvinKirs** [comment]
Hi, please resolve the conflict..thx

**Comment by Segun-Ogundipe** [comment]
> Hi, please resolve the conflict..thx

Done @CalvinKirs

**Review by Tianqi-Dotes — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**